analyzers/src/SonarAnalyzer.CSharp/SymbolicExecution/Roslyn/PublicMethodArgumentsShouldBeCheckedForNull.cs

```diff
@@ -93,13 +93,14 @@ protected override ProgramState PreProcessSimple(SymbolicContext context)
 
     private static bool IsParameterDereferenced(IOperationWrapperSonar operation) =>
         operation.Parent != null
-        && operation.Parent.IsAnyKind(
+        && (operation.Parent.IsAnyKind(
```

Comment:
This condition is getting a bit difficult to read. 
Maybe it would make sense to build an array of `OperationKindEx`, to be used in `IsAnyKind`?